Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorPaintEffect.cxx

```diff
@@ -177,6 +187,7 @@ class BrushPipeline2D : public BrushPipeline
   vtkSmartPointer<vtkTransformPolyDataFilter> BrushWorldToSliceTransformer;
   vtkSmartPointer<vtkCutter> FeedbackCutter;
   vtkSmartPointer<vtkTransformPolyDataFilter> FeedbackWorldToSliceTransformer;
+  vtkSmartPointer<vtkCallbackCommand> ErrorSinkCallbackCommand;
```

Comment:
Won't this mask other potentially meaningful errors?

Reply:
The filter does not provide any meaningful error messages. It logs an error if there are no points in the input polydata (which is plain wrong) or if the input transform is not defined (this might be marginally useful, but still unnecessary, as the filter could simple interpret "no transformation is specifed" as "no transformation should be performed).